v2/pkg/passive/sources.go

```diff
@@ -39,185 +46,102 @@ import (
 	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping/sources/zoomeyeapi"
 )
 
-// DefaultSources contains the list of fast sources used by default.
-var DefaultSources = []string{
-	"alienvault",
-	"anubis",
-	"bufferover",
-	"c99",
-	"certspotter",
-	"censys",
-	"chaos",
-	"chinaz",
-	"crtsh",
-	"dnsdumpster",
-	"fofa",
-	"fullhunt",
-	"hackertarget",
-	"intelx",
-	"passivetotal",
-	"robtex",
-	"riddler",
-	"securitytrails",
-	"shodan",
-	"threatcrowd",
-	"threatminer",
-	"virustotal",
-	"whoisxmlapi",
+var AllSources = [...]subscraping.Source{
+	&alienvault.Source{},
+	&anubis.Source{},
+	&archiveis.Source{},
+	&bevigil.Source{},
+	&binaryedge.Source{},
+	&bufferover.Source{},
+	&c99.Source{},
+	&censys.Source{},
+	&certspotter.Source{},
+	&chaos.Source{},
+	&chinaz.Source{},
+	&commoncrawl.Source{},
+	&crtsh.Source{},
+	&dnsdb.Source{},
+	&dnsdumpster.Source{},
+	&fofa.Source{},
+	&fullhunt.Source{},
+	&github.Source{},
+	&hackertarget.Source{},
+	&intelx.Source{},
+	&passivetotal.Source{},
+	&rapiddns.Source{},
+	&riddler.Source{},
+	&robtex.Source{},
+	&securitytrails.Source{},
+	&shodan.Source{},
+	&sitedossier.Source{},
+	&sonarsearch.Source{},
+	&threatbook.Source{},
+	&threatcrowd.Source{},
+	&threatminer.Source{},
+	&virustotal.Source{},
+	&waybackarchive.Source{},
+	&whoisxmlapi.Source{},
+	&zoomeye.Source{},
+	&zoomeyeapi.Source{},
 }
 
-// DefaultRecursiveSources contains list of default recursive sources
-var DefaultRecursiveSources = []string{
-	"alienvault",
-	"binaryedge",
-	"bufferover",
-	"certspotter",
-	"crtsh",
-	"dnsdumpster",
-	"hackertarget",
-	"passivetotal",
-	"securitytrails",
-	"sonarsearch",
-	"virustotal",
-}
+var NameSourceMap = make(map[string]subscraping.Source, len(AllSources))
```

Comment:
the make statement should be a part of the init function instead of being called globally